module_assays/docs/docs.py

```diff
@@ -0,0 +1,19 @@
+import mongoengine as me
+import datetime
+import shortuuid
+
+class Assays(me.DynamicDocument):
```

Comment:
The collection is called Assays, it needs to be the same as the collection name I believe. Its weird because to define a single assays you would do `assay = Assays()` but to query you do `Assays.objects(condition=condition)` 

We will have to either leave it as Assays or change it to Assay and change the collection name to Assay (which it is called binding as of now so it needs to be changed anyway)

Reply:
given that the construct we'll work within the code is the `Assay/s` class and that a single instance represents only a single assay, logical consistency within the code should take precedence over what makes sense from a database naming perspective. No one should actually be interacting with the database backend